packages/common/src/hardhat/tasks/identifiers.js

```diff
@@ -72,11 +94,17 @@ task("migrate-identifiers", "Adds all whitelisted identifiers on one IdentifierW
           const receipt = await newWhitelist.methods.addSupportedIdentifier(identifiersToWhitelist[i]).send({
             from: deployer,
           });
+          console.log(
+            `${i}: Added new identifier ${web3.utils.hexToUtf8(identifiersToWhitelist[i])} (${receipt.transactionHash})`
+          );
           addSupportedIdentifierReceipts.push(receipt);
+        } else {
+          // Explicitly push `null` so that `txn` and `identifier` line up in table to print to console.
+          addSupportedIdentifierReceipts.push({ transactionHash: "Already whitelisted" });
         }
       }
       addSupportedIdentifierReceipts.forEach((receipt, i) => {
-        resultsTable[i] = { ...resultsTable[i], txn: receipt.transactionHash };
+        resultsTable[i] = { ...resultsTable[i], txn: receipt?.transactionHash };
```

Comment:
nit: why can't we just add this to results table in the loop above?

Reply:
good point... done